src/store/accounts/index.ts

```diff
@@ -137,6 +151,15 @@ export const currentAccountDataState = atom(get => {
   return get(accountDataResponseState([address, network.url]));
 });
 
+export const currentAccountAvailableStxBalanceState = atom(get => {
+  const currentAccountData = get(currentAccountDataState);
+  if (!currentAccountData) return;
+  const stx = new BigNumber(currentAccountData.balances.stx.balance);
+  const lockedStx = new BigNumber(currentAccountData.balances.stx.locked);
+  const curBalance = stx.minus(lockedStx);
+  return curBalance.toString();
```

Comment:
Same here, better to call toString() when consumed, than have to reinitialise another BigNumber

Reply:
Ok, yep, I debated about which was better. I will change it, thx for the feedback!